### PR TITLE
Hw-1

### PR DESCRIPTION
**Pull request by Kaijou88**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1 && result2 == bet2) ? 2 :
+                ((result1 == result2 && bet1 == bet2) ? 1 :
+                    ((result1 < result2 && bet1 < bet2) ? 1 :
+                        ((result1 > result2 && bet1 > bet2) ? 1 : 0)));
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: simplify to write "? 1" only once

Comment (Tarabarnik) [reply src/main/java/core/basesyntax/FootballMatch.java]: write as few brackets () as possible :)

Comment (Kaijou88) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2 : (result1 == result2 && bet1 == bet2)
+                || (result1 < result2 && bet1 < bet2) || (result1 > result2 && bet1 > bet2) ? 1 : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        return (result1 == bet1 && result2 == bet2) ? 2 : 
        (result1 == result2 && bet1 == bet2) || 
        (result1 < result2 && bet1 < bet2) || 
        (result1 > result2 && bet1 > bet2) ? 1 : 0;
```
the code should be written in one style and easy to read

Comment (Kaijou88) [reply src/main/java/core/basesyntax/FootballMatch.java]: ok, thank you,

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**